schema/samples/access-policy.json

```
@@ -0,0 +1,26 @@
+{
+  "@type": "Set",
```

```suggestion
  "@type": "Set",
  "@id": "some-id"
```

According to the schema policies should contain `@id`